feat(curriculum):add tasks for ES curriculum A2 Block 2 (#60768)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/26-a2-professional-spanish/get-to-know-others-by-asking-simple-questions/6840d8ad9dd41ad753e7d03d.md

---
id: 6840d8ad9dd41ad753e7d03d
title: Task 6
challengeType: 19
dashedName: task-6
---

<!-- (Audio) Marisol: Hola Mateo, estoy bien, gracias. Un poco cansada. -->

# --instructions--

Listen to the audio and answer the question below.

# --questions--

## --text--

> How does Marisol first respond to Mateo's question?

Reviewer: I found this question a bit confusing at first. 

When the question mentions "first respond", I thought it was referring to the entire question, including `cansada` and then I realized that it was only the first part of the sentence, where she mentions that she is fine. 

To simplify this, I would suggest:

```suggestion
What is the first thing Marisol says in response to Mateo's question about how she is?
```

## --answers--

> She says she is bored

Reviewer: ```suggestion
She says she is bored.
```

### --feedback--

Marisol does not say she is bored.

---

> She says she is fine

Reviewer: ```suggestion
She says she is fine.
```

---

> She says she is busy

Reviewer: ```suggestion
She says she is busy.
```
